game.py

```diff
@@ -20,7 +21,31 @@ def snowman(snowman_word):
     If the player wins and, 
     'Sorry, you lose! The word was {snowman_word}' if the player loses
     """
-    pass
+
+    wrong_guesses_list = []
+    correct_letter_guess_statuses = build_letter_status_dict(snowman_word)
+
+    while len(wrong_guesses_list) < SNOWMAN_MAX_WRONG_GUESSES:
+        user_input = get_letter_from_user(correct_letter_guess_statuses, wrong_guesses_list)
+
+        if user_input in snowman_word:
+            print("You guessed a letter that's in the word!")
+            #log correct letter guess
+            correct_letter_guess_statuses[user_input] = True
+            #check if complete
+            if is_word_guessed(snowman_word, correct_letter_guess_statuses):
+                print(f"Congratulations, you win!")
+                return
```

Comment:
Love that you are utilizing the `return` keyword! It allows you to not only terminate the loop but the function entirely. Something to consider here is readability, specifically in two places: 
1. Your current `while` statement is `while len(wrong_guesses_list) < SNOWMAN_MAX_WRONG_GUESSES:`. This communicates that your loop termination condition is the user exceeding the number of allowed guesses. Though, here you have another termination condition nested in the loop. It could be overlooked or not interpreted as readily. 
2. You have 3 levels of indention for this condition. You will soon see that conditional blocks that are deeply nested can lower the readability of your code as well. How could we refactor the code to remove a layer or two of nesting.